doc/source/user_guide/index.rst

```diff
@@ -22,4 +23,5 @@ While the Ansys Sphinx theme is often used as is, you can customize the followin
    options
    pdf
    404_page
+   link_code
```

Comment:
Let's better create a `Extensions` page and include the `link_code` in here.

Reply:
Agreed! The ``Extensions`` section will be much more scalable